src/main/java/commands/SearchCommand.java

```diff
@@ -87,16 +124,36 @@ public void searchExercise() {
         }
     }
 
-    public boolean isMatchExercise(String exercise) {
+    public boolean isMatch(String exercise) {
         return exercise.contains(userArguments);
```

Comment:
I think the parameter name "exercise" should be changed as this method is more generalized now?

Reply:
yes, thank you for the reminder!